### PR TITLE
fix: Search by number not working

### LINKED ISSUE #8793: Search by number not working
Generative drop NFTs have numbers #1 #2 #3 ... why Im not able to search by number?

AHP, production, 
Genera collection: https://kodadot.xyz/ahp/collection/63
Tried number 4 I tried to search it with # - not working

Screenshot 2023-12-28 at 17 33 25

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #8793

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2023-12-29 at 11 31 50

### CODE REVIEW

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

we should still check that the search behavior does not change

**Comment by roiLeo** [comment]
breaking tests not related?

File: components/items/ItemsGrid/utils/useSearchParams.ts [hunk 1]
```diff
@@ -5,9 +5,12 @@ function useSearchKeywords() {
 
   return {
     keywords: computed(() => {
-      return route.query.search?.length
-        ? [{ name_containsInsensitive: route.query.search }]
-        : []
+      const search = route.query.search
+      return search?.length
+        ? {
+            OR: [{ name_containsInsensitive: search }, { sn_contains: search }],
```
Comment (Jarsen136) [inline components/items/ItemsGrid/utils/useSearchParams.ts:11]: Only if the search text is number, we could add it to this search params `sn`

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
thanks!
pay 20 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $20 @ 8.37 USD/DOT ~ 2.389 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xce3b7db95f828d6ddc2e1233795ff3c1572270cef8295dae4016c10423a1f4c8](https://polkadot.subscan.io/extrinsic/0xce3b7db95f828d6ddc2e1233795ff3c1572270cef8295dae4016c10423a1f4c8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**